src/NewBookMarkForm/NewBookMarkForm.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
+
+export const NewBookMarkForm = () => {
+    const [minutesRead, setMinutesRead] = useState(0);
+    const [endingPage, setEndingPage] = useState(0);
+    const [notes, setNotes] = useState('');
+    const [reactions, setReactions] = useState('');
+
+    return(
+        <form>
+          <label>
+          How many minutes did you read?
+            <input
+              aria-label="reading minutes input"
+              className="reading-minutes-input"
+              placeholder="minutes spent reading"
+              value={minutesRead}
+              onChange={event => setMinutesRead(event.target.value)}>
+            </input>
+          </label>
+          <label>
+          What page did you end on?
+            <input
+              aria-label="ending page input"
+              className="ending-page-input"
+              placeholder="page number of the last page you read"
+              value={endingPage}
+              onChange={event => setEndingPage(event.target.value)}>
+            </input>
+          </label>
+          <label>
+          Any thoughts or notes?
+            <input
+              aria-label="notes input"
+              className="notes-input"
+              placeholder="What did you think of what you read?"
+              value={notes}
+              onChange={event => setNotes(event.target.value)}>
+            </input>
+          </label>
+          <label>
+          Reactions?
+            <input
+              aria-label="reactions-input"
```

Comment:
may want this to be a selection of emojis rather than a blank input (the intent of this PR is just to lay out the basic structure since we may implement a specific form library)